tests/data/btf_data.h

```diff
@@ -58,10 +58,16 @@
 //    struct file *file;
 //  };
 //
+//  struct bpf_iter__task_vma {
+//    struct task_struct *task;
+//    struct vm_area_struct *vma;
+//  };
+//
```

Comment:
Updating the comment is not sufficient, you have to re-generate the BTF data. See below for commands.

Reply:
Finish it, thanks.

```bash
 $ cat a.c                                                                   
 [... the above definitions ...]                                             
 $ gcc -o btf -g btf.c                                                       
 $ pahole -J btf   # v1.17 and higher                                        
 $ objcopy --dump-section .BTF=data btf                                      
 $ xxd -i data > data.h
```